docs/viz/overview.rst

```diff
@@ -324,6 +324,15 @@ Altair seit mindestens einem Jahr nicht mehr weiterentwickelt wurden (Stand:
     "`toyplot <https://github.com/sandialabs/toyplot>`_",".. image:: https://raster.shields.io/github/stars/sandialabs/toyplot",".. image:: https://raster.shields.io/github/contributors/sandialabs/toyplot",".. image:: https://raster.shields.io/github/commit-activity/y/sandialabs/toyplot",".. image:: https://raster.shields.io/github/license/sandialabs/toyplot"
     "`ipyleaflet <https://github.com/jupyter-widgets/ipyleaflet>`_",".. image:: https://raster.shields.io/github/stars/jupyter-widgets/ipyleaflet",".. image:: https://raster.shields.io/github/contributors/jupyter-widgets/ipyleaflet",".. image:: https://raster.shields.io/github/commit-activity/y/jupyter-widgets/ipyleaflet",".. image:: https://raster.shields.io/github/license/jupyter-widgets/ipyleaflet"
 
+Anmerkungen zu Lizenzen laut Projektangaben:
+
+* Apache 2.0: bqplot, TensorFlow, Yellowbrick
+* BSD 2.0 : altair, Bokeh, d3po, geopandas, ggpy, glumpy, holoviews, mpld3, networkx, NumPy, Pandas, pythreejs, scikit-learn, SciPy, Seaborn, Statsmodels, SymPy, Theano, vega, VisPy 
```

Comment:
* altair, Bokeh, d3po, glumpy, holoviews, mpld3, networkx, NumPy, Pandas , pythreejs, scikit-learn, SciPy, Seaborn, vega and VisPy appears to be released under the BSD 3-Clause License, see https://github.com/bokeh/bokeh/blob/master/LICENSE.txt, https://github.com/altair-viz/altair/blob/master/LICENSE, https://github.com/adrn/d3po/blob/master/LICENSE.txt, https://github.com/geopandas/geopandas/blob/master/LICENSE.txt, https://github.com/glumpy/glumpy/blob/master/LICENSE.txt, https://github.com/holoviz/holoviews/blob/master/LICENSE.txt, https://github.com/mpld3/mpld3/blob/master/LICENSE, https://github.com/networkx/networkx/blob/master/LICENSE.txt, https://github.com/numpy/numpy/blob/master/LICENSE.txt, https://github.com/pandas-dev/pandas/blob/master/LICENSE, https://github.com/jupyter-widgets/pythreejs/blob/master/LICENSE, https://github.com/scikit-learn/scikit-learn/blob/master/COPYING, https://github.com/scipy/scipy/blob/master/LICENSE.txt, https://github.com/mwaskom/seaborn/blob/master/LICENSE, https://github.com/vega/vega/blob/master/LICENSE and https://github.com/vispy/vispy/blob/master/LICENSE.txt
* Statsmodels, SymPy and Theano  are currently not discussed in the Jupyter tutorial